libs/feature-workspaces/src/lib/new-workspace/new-workspace.component.scss

```diff
@@ -1,77 +1,32 @@
 angular-console-new-workspace {
-  width: 100%;
-  overflow: hidden;
-
-  .error {
-    background-color: red;
-  }
-
-  .ui-flags {
-    background: white;
-    overflow: auto;
-
-    .name {
-      padding: 24px;
-      font-size: 20px;
-      font-weight: 500;
-    }
-
-    .description {
-      padding: 24px;
-    }
+  .mat-vertical-content {
+    padding-bottom: 0 !important;
   }
 
-  .submit-button-container {
-    margin: 40px 20px;
-  }
-
-  .mat-expansion-panel-content {
-    max-height: calc(100vh - 278px) !important;
-
-    overflow: auto !important;
+  .path-form-field,
+  .path-form-field input {
+    cursor: pointer;
   }
 
-  .expand .mat-expansion-panel-content {
-    max-height: calc(100vh - 230px) !important;
+  .selected-schematic {
+    background: #dcdcdc !important;
```

Comment:
It'd be good for us to define a set of app css variables, so we can keep things consistent. No need to do it now since it could be a cleanup task later.

Reply:
At the very least I'm extracting the colors out to variables. We can move those variables to a common file in a clean up :)